dev/generate_bom.py

```diff
@@ -83,6 +85,8 @@ def init_argument_parser(cls, parser):
                         help="Docker registry to push the container images to.")
     parser.add_argument('--changelog_output', default='',
                         help="Output file to write the changelog to.")
+    parser.add_argument('--nightly', default=False, action='store_true',
```

Comment:
To be clear here, the flag will accept any string as the alias before '.yml'. We'll use this to do our 'nightly.yml' for the time being. Later this may exist elsewhere.